packages/cli/generation/ir-generator/src/generateIntermediateRepresentation.ts

```diff
@@ -54,31 +54,33 @@ import { generateEndpointExample } from "./examples/generator/generateSuccessEnd
 import { convertIrToDynamicSnippetsIr } from "./dynamic-snippets/convertIrToDynamicSnippetsIr";
 
 export async function generateIntermediateRepresentation({
-    fdrApiDefinitionId,
     workspace,
     generationLanguage,
     keywords,
     smartCasing,
     disableExamples,
-    includeOptionalRequestPropertyExamples,
     audiences,
     readme,
     packageName,
     version,
-    context
+    context,
+    fdrApiDefinitionId,
+    includeOptionalRequestPropertyExamples,
+    sourceResolver = new NopSourceResolver()
```

Comment:
@dsinghvi this will be fixed before merge - if everything else look good I'll make this a required property so that every caller needs to explicitly pass it in themselves. It's otherwise too easy for us to miss it.

Reply:
Changed to a required property in [this commit](https://github.com/fern-api/fern/pull/5295/commits/62c507e319827b52a1d5c37bbed57a54968e7452).